engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -372,6 +373,15 @@ def has_no_regcode?(auth_header)
         protected
 
         def scc_activate_product
+          if (@system.system_information &&
+              JSON.parse(@system.system_information)['cloud_provider'].casecmp('microsoft').zero? &&
+              @product.product_class.downcase.include?('ltss') &&
+              InstanceVerification.provider.new(logger, request, nil, @system.instance_data).basic?
```

Comment:
Thinking ahead this should maybe be more generic, i.e. the verification code should probably have `allows_ltss?` rather than `basic`. If at some point other addons are made available then we can simply extend the logic to be product specific rather than having the function name a specific designation for the listing.